practice/1_python_part_1/task2.py

```diff
@@ -13,4 +13,14 @@
 
 
 def set_to_dict(dict_to_update: Dict[str, int], **items_to_set) -> Dict:
-    ...
+    if not bool(dict_to_update):
+        dict_to_update = items_to_set
+        return dict_to_update
+    elif not bool(items_to_set):
+        return dict_to_update
+    else:
+        for key in dict_to_update:
+            if key in items_to_set:
+                if dict_to_update[key] < items_to_set[key]:
+                    dict_to_update[key] = items_to_set[key]
+                    return dict_to_update
```

Comment:
since dictionaries are mutable, there is no need to return them
also `bool` operators are redundant, you can just omit them and go with `if not dict_to_update` and so on